corehq/toggles.py

```diff
@@ -439,6 +439,13 @@ def _ensure_valid_randomness(randomness):
     [NAMESPACE_DOMAIN],
 )
 
+OPTIONAL_MEDIA_IN_UPDATES = StaticToggle(
```

Comment:
the name of this and the feature are different, I see people searching for "lazy video" and then not finding it. You might want to reconsider this name

Reply:
I used this name from the spec file. 
But you are right, it would be confusing for uses.
I would update it to `LAZY_LOAD_VIDEOS` and will update in the specs

Reply:
@mkangia  @AmitPhulera The property `LAZY_LOAD_VIDEOS` is just one setting under the feature that makes all videos in an application optional, though the feature flag is aimed to provide cover for all such settings we roll out in future. If we have to make a change here, I think changing the property name to `mark-all-videos-optional` is a better change here instead of changing the feature flag.

Reply:
Okay. Going with `mark-all-videos-optional`

Reply:
@shubham1g5 Can we go with `LAZY_LOAD_MULTIMEDIA` feature flag, I belive it can encompass any other change that we do in the future. 

The setting name `mark-all-videos-optional` gives me a sense that we are making videos optional which we are not. They will eventually get downloaded to the app.

What are your views on it @mkangia @orangejenny ?

Reply:
no objections to `LAZY_LOAD_MULTIMEDIA` from my side.